geometry_msgs/Pose pose
bool franka_gripper